artifacts/rhdh-plugins-reference/argocd/argocd-plugin-admin.adoc

```diff
@@ -66,11 +66,13 @@ global:
     plugins:
       - package: ./dynamic-plugins/dist/roadiehq-backstage-plugin-argo-cd-backend-dynamic
         disabled: false
-      - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-argocd
+      - package: ./dynamic-plugins/dist/backstage-community-plugin-redhat-argocd
         disabled: false
 ----
 
 [role="_additional-resources"]
 .Additional resources
 
-* For more information on dynamic plugins, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/configuring_plugins_in_red_hat_developer_hub/rhdh-installing-dynamic-plugins[Dynamic plugin installation].
+* Note that the package path, scope, and name of the Red Hat ArgoCD plugin has changed since 1.2. For more information, see link:{LinkReleaseNotes}#removed-functionality-rhidp-4293[Breaking Changes] in the _{rn-product-title}_. 
+
+* For more information on dynamic plugins, see link:{LinkPluginsGuide}#rhdh-installing-dynamic-plugins[Dynamic plugin installation] in the _{NameOfPluginsGuide}_ guide.
```

Comment:
The added link returns 404. I'd suggest adding the following link:
```suggestion
* For more information on dynamic plugins, see link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/installing_and_viewing_dynamic_plugins/index[Installing and viewing dynamic plugins]
```

@Gerry-Forde We need to also include the attribute URLs for the new plugin titles to avoid using hardcoded links.

Reply:
@hmanwani-rh I've also added a comment/suggestion for this ^^^